_grades/A10.md

```diff
@@ -0,0 +1,117 @@
+
+### Technical Assessment and Adoption
+
+You identify and assess business change situations and the technical solutions that can support a client organization, and define its governance processes and program management
+
+You can assess market trends, ensuring your team has all the modern skills required to deliver high quality products, and outperform competing companies
+
+### Business Awareness
+
+You demonstrate a passion for our business and constant hunger for outstanding performance
+
+You understand and analyze the complexities and opportunities inherent in a global business environment
+
+You understand the political infrastructure of board level situations and operate accordingly
+
+You create and drive business plans for your area
+
+You have completed the required finance and commercial courses for this grade
+
+### Developing Our Team
+
+You understand how we make money and how you and your team or decisions fit into that
+
+You lead your team to strive to drive profitable growth and exceed expectations 
+
+You implement effective recruitment strategies to build the organisation and meet business needs
+
+### Winning New Work
+
+You create opportunities to develop markets/clients and adapt our business
+
+You devise and set the business strategy to achive market strength and business growth
+
+You have advocated wins in the market, leveraging your high expertise and personal insights
+
+You constantly bring value to widen positions in a sustainable way and build long-term partnerships with clients
+
+You influence and develop the future direction of our client base
+
+## Team Leadership
+
+You are an engagement lead and a driving force in the partnership between Capgemini and the client
+
+You lead and motivate multi-functional teams in dispersed or complex environments
+
+You embody the values of Capgemini, and convey the company vision to your teams
+
+You suggest and lead new initiatives both within the team and the wider company
+
+You have substantial knowledge of the wider Group's capabilities, and encourage co-operative team working across the business, to break silos and leverage the best teams, expertise and assets
+
+You take a high degree of responsibility and ownership of people issues within your area
+
+You boost teams’ and individuals’ potential to improve their engagement, performance, and career experience
+
+Your teams work in an agile way, with efficiency, speed, and simplification in all actions
+
+## Advocacy and Ambassadorship 
+
+You are acknowledged internally and externally as a thought leader
+
+You are seen as a role model in your community and provide leadership in more than one specific area
+
+You can operate effectively at the highest levels within national and multi-national organisations
+
+You are a change ambassador of our transformation, expanding your networks both inside and outside the company to bring value
```

Comment:
Maybe this is copied from somewhere, but "a change ambassador of our transformation" doesn't make sense to me. What is our transformation?

Reply:
Yeah I have got some of the wording from different sources. I have re-worded to make it clearer :-)